packages/cli/src/push/abstract.push.ts

```diff
@@ -16,16 +17,20 @@ export interface AbstractPushEvents {
  *
  * @emits message when a message is received from a client
  */
+@Service()
 export abstract class AbstractPush<Connection> extends TypedEmitter<AbstractPushEvents> {
 	protected connections: Record<string, Connection> = {};
 
 	protected userIdByPushRef: Record<string, string> = {};
 
 	protected abstract close(connection: Connection): void;
 	protected abstract sendToOneConnection(connection: Connection, data: string): void;
+	protected abstract ping(connection: Connection): void;
 
 	constructor(protected readonly logger: Logger) {
 		super();
+		// Ping all connected clients every 60 seconds
+		setInterval(() => this.pingAll(), 60 * 1000);
```

Comment:
Should probably clear this interval in `closeAllConnections`

Reply:
[this has been copied over from the websocket code](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/push/websocket.push.ts#L20). We could definitel clean it up, but it's clearly not an issue considering that it's been around like this since 2 years.
I'll fix this in the next PR.